packages/react-components/src/localization/translated/fr_FR.json

```diff
@@ -58,7 +58,9 @@
     "wednesday": "Mercredi",
     "thursday": "Jeudi",
     "friday": "Vendredi",
-    "saturday": "Samedi"
+    "saturday": "Samedi",
+    "participantJoined": "rejoint le chat.",
+    "participantLeft": "a quitté le chat."
   },
   "errorBar": {
     "unableToReachChatService": "Tu es hors ligne",

```

Comment:
'Vous êtes hors ligne.'

Reply:
Who needs a Touchdown service when you have an Aline! 😁